docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+- New projects are created with a default priority level and are not active
+- Projects do not have a start or end date (start or end dates are specified on allocations, users and attributions)
+- New projects must have a name, description, project owner and institution
+ 
+### Project administration
+- Principal investigators can associate new/existing users to an existing projects by email address
+- Associating users to a project should fail after a specified number of user has been reached (to avoid excessive disk space to a project)
+- The associated user limit should be set on a per-project basis and have a per-institution limit
+- Users associated with a project should be required to click a link in a confirm email before being added
+- Project owner can request a raised priority level if priority criterion is met (requests are sent to a system administrator to handle manually)
+
+### User creation
+- Anyone with an institutional email address should be able to sign up for a user account
+- Anyone with a pre-approved (non-institutional) email address can create an account
+- Users should have a name, email address, institution, college/department, orcid, scopus url and homepage
+- Users from Swansea university must have a Cronfa URL
+- Users should be automatically removed from the system when they leave university
+- User signup should require agreeing to terms and conditions
+ 
+### Attribution addition and approval
+- Users can add an attributions to a projects they are associated, specifying an attribution owner (e.g. principal investigator or lead author)
+- An attribution can be of type grant or paper
+- An attribution can associated with one or more projects
+- A paper has a title, authors, a Journal, a doi and a cronfa link (Swansea)
+- A grant has a title, grant code, total amount, amount attributed to SCW, grant code, comments, a funding body and a principal investigator
+- Principal investigator receives a confirmation request email for every grant application
+- Confirmation request emails contain a mailto link to an email reply statement, pre-populated with information derived from the attribution data and confirming that the sender is PI of the grant and that grant income is only attributed once.
+- Confirmation request emails will tell the user to modify the text if incorrect
+- Attribution authoriser is notified of a new attribution at their institution
+- Attribution authoriser can mark an attribution confirmation email as valid
+- Attribution authoriser can see highlighted changes to attribution confirmation emails (compared to original message)
+
+### Project approval and system allocation
+- Project approvers, RSEs and attribution authorisers can see a page listing pending/unapproved projects per institution
+- Project approvers, RSEs and attribution authorisers can see details of pending/unapproved projects per institution
+- Project approvers can set project status to approved
```

Comment:
Existing roles of RSE and Sysadmins can.

Reply:
We could do this, I would suggest that we create a group which has the permission to approve projects. The permission group could then be assigned to any user as and when required. It's a lot easier to enable and disable features via permissions rules.

Reply:
Seems like a reasonable approach. I'd say this one might be worth trying to push into the first release if possible, on the basis that it'll be harder to change later.